src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRent = 40;
+  let total;
+
+  if (days < 3) {
+    total = dailyRent * days;
+  } else {
+    if (days >= 3 && days < 7) {
+      total = (dailyRent * days) - 20;
```

Comment:
When the code comes to this else, we know that ```!(days < 3) or (days >= 3) same thing``` so you can omit that first condition on line 37:

```js
  } else if (days < 7) { // lline36 and line37 combined

```

Reply:
Done!